scripts/test-go.sh

```diff
@@ -15,13 +19,19 @@ else
     exit 1
 fi
 
-echo ">> Testing core"
-cd ${CORE_PATH}
-go test ./... ${GOTEST_ARGS}
+if [ ! "${SKIP_NON_GENERATED_FILES}" = true ]; then
```

Comment:
Could we safely adjust the condition? to be `!= true` or flip it to be `= false`?

Reply:
`!= true` is what is currently doing.

You can't flip it to `= false` because bash is not forcing this to be `true` or `false`. So if you were to pass `"banana"`, `"${SKIP_NON_GENERATED_FILES}" = true` is false and `"${SKIP_NON_GENERATED_FILES}" = false` is also false.

I've added input cleanup at the top to make the script clearer, PTAL